### ISSUE TITLE
[Bug] GlowTTS training bug 

### ISSUE THREAD

**Issue by erogol** [coqui-ai/TTS#813]
### Discussed in https://github.com/coqui-ai/TTS/discussions/810

<div type='discussions-op-text'>

<sup>Originally posted by **jefke24** September 15, 2021</sup>
Hello,

I'm just wondering if the behaviour I'm seeing is normal in tensorboard.
I'm new at this and I'm finetuning the glowtts model based on a new dataset, following the instructions here https://tts.readthedocs.io/en/latest/finetuning.html (nice job by the way in adding these instructions recently, just what I was looking for).

At first, in the first few epochs, I'm seeing this in tensorboard regarding groundtruth and prediction:

![image](https://user-images.githubusercontent.com/90794131/133487422-2bf63868-47c8-456e-aee8-203b39b5ac5f.png)

So at least I'm kind of recognizing the groundtruth in the prediction graph.
However, after about 150 epochs with 11 steps each, I end up with this:

![image](https://user-images.githubusercontent.com/90794131/133487551-9d72d7a5-685c-4eaf-b6ba-9c67cc3255f5.png)

Obviously the audio is complete noise at this point.

As for evolution of loss etc.:

![image](https://user-images.githubusercontent.com/90794131/133487666-2ac7b70c-a8ab-4e34-a17a-896722f427ee.png)

So if I'm reading this correctly, loss is going down and converging (however keeps getting stuck around 1.0, not going down anymore).
Attention graph is going towards a diagonal, which is the desired outcome I believe?

But prediction and ground truth are completely deviating.
I'm wondering: is this expected behaviour when finetuning a pre-trained model?
With enough training/epochs, can I expect groundtruth and prediction to converge again?

Or as soon as you're seeing this kind of behaviour, it's basically done and you need to start over with other parameters and/or better data?

Thanks in advance for any feedback you can give me!</div>
